package NDnnProto;

message TChronotronRuleConst {
}

message TChronotronRuleState {
}